translate/src/modules/user/components/UserNotification.tsx

```diff
@@ -11,6 +11,38 @@ type Props = {
   notification: Record<string, any>;
 };
 
+interface DateDisplayProps {
+  date: string;
+  date_iso: string;
+}
+
+const DateDisplay: React.FC<DateDisplayProps> = ({ date, date_iso }) => {
+  const formattedDate: string = new Date(date).toLocaleDateString('en-US', {
+    day: 'numeric',
+    month: 'long',
+    year: 'numeric',
+  });
+  const isOld: boolean =
+    new Date().getTime() - new Date(date).getTime() > 7 * 24 * 60 * 60 * 1000;
```

Comment:
I haven't test it, but I find the python logic (`is_old_notification()`) easier to read. Thoughts?

```suggestion
    new Date(date).getTime() < new Date().getTime() - 7 * 24 * 60 * 60 * 1000;
```

Reply:
I'm fine either way, although I wrote it originally because I wanted to read it left-to-right like
Current Date minus Notification Date is greater than 7 days.

Reply:
Other than that, I'll commit the change cuz this way is perfectly understandable.